packages/mds-agency/request-handlers.ts

```diff
@@ -180,7 +179,11 @@ export async function updateVehicleFail(
   } else if (!provider_id) {
     res.status(404).send({})
   } else {
-    logger.error(`fail PUT /vehicles/${device_id}`, { providerName: providerName(provider_id), body: req.body, error })
+    AgencyLogger.error(`fail PUT /vehicles/${device_id}`, {
+      providerName: providerName(provider_id),
```

Comment:
We should stop using the `providerName` function so we can eventually get rid of `mds-providers` altogether in favor of an async data source.